docs/source/getting-started.md

```diff
@@ -23,9 +23,11 @@ iwr 'https://raw.githubusercontent.com/apollographql/rover/v0.0.1/installers/bin
 
 Rover is distributed on npm for easy integration with your JavaScript projects.
 
-##### devDependency install
```

Comment:
Note I made this adjustment because there is no property called `devDependency`, but rather, `devDependencies`.